.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] As stated the project requirement Kindly make sure that Added `node_modules` to `gitignore` file, when you Set up [a linter for HTML and CSS](https://github.com/microverseinc/linters-config) and make sure that any files that are not relevant are ignored by git.

Reply:
Fixed.